dbus_service.py

```diff
@@ -515,7 +538,7 @@ def _update_index(self):
     def get_values_for_inverter(self):
         '''read data and return (power, pvyield, current, voltage)'''
         meter_data = self._get_data()
-        (power, pvyield, current, voltage) = (None, None, None, None)
+        (power, pvyield, current, voltage, dc_voltage) = (None, None, None, None, None)
```

Comment:
Some day we have to introduce some other way of handing these multiple values... (new transport class?)

Reply:
I totally agree but for now it should do it